Packs/Zendesk/Integrations/Zendeskv2/Zendeskv2.py

```
@@ -679,178 +817,184 @@ def zendesk_article_list(self, locale: Optional[str] = '', article_id: Optional[
-    #     zendesk_ticket_scheme.add_field(name='recipient', description='the description for the field')
-    #     zendesk_ticket_scheme.add_field(name='requester', description='the description for the field')
-    #     zendesk_ticket_scheme.add_field(name='assigne', description='the description for the field')
-    #     zendesk_ticket_scheme.add_field(name='organization', description='the description for the field')
-    #     zendesk_ticket_scheme.add_field(name='collaborators', description='the description for the field')
-    #     zendesk_ticket_scheme.add_field(name='followers', description='the description for the field')
-    #     zendesk_ticket_scheme.add_field(name='email_ccs', description='the description for the field')
-    #     zendesk_ticket_scheme.add_field(name='tags', description='the description for the field')
-    #     zendesk_ticket_scheme.add_field(name='custom_fields', description='the description for the field')
-    #     zendesk_ticket_scheme.add_field(name='followup_ids', description='the description for the field')
-
-    #     return_results(GetMappingFieldsResponse([zendesk_ticket_scheme]))
-
-
-def main():
+        demisto.incidents(tickets)
+        demisto.setLastRun(created_tickets.next_run())
+
+    def get_modified_remote_data(self, lastUpdate: Optional[str] = None):
+        try:
+            timestamp = int(dateparser.parse(lastUpdate).timestamp())  # type: ignore
+        except (TypeError, AttributeError):
+            timestamp = 0
+        last_run = get_last_mirror_run() or {}
+        updated_tickets = UpdatedTickets(self, timestamp, last_run)
+        tickets_ids = list(map(lambda x: str(x['id']), updated_tickets.tickets()))
+        if tickets_ids:
+            return_results(GetModifiedRemoteDataResponse(tickets_ids))
+        try:
+            set_last_mirror_run(updated_tickets.next_run())
+        except json.decoder.JSONDecodeError:
+            pass
```

never silently ignore an error, add a log